### PR TITLE
feat: update seata version to 2.0.0 in 2023.x

### PR DESCRIPTION
**Pull request by yuluo-yx**
### Describe what this PR does / why we need it

- Updated seata version to 2.0.0 in 2023.x;
- For https://github.com/alibaba/spring-cloud-alibaba/issues/3574.

### CODE REVIEW

File: spring-cloud-alibaba-examples/seata-example/account-service/src/main/resources/application.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ base:
       dbname: seata #your database name for test
       port: 3306 #your mysql server listening port
       username: 'root' #your mysql server username
-      password: 'root' #your mysql server password
+      password: '082916' #your mysql server password
```
Comment (ruansheng8) [inline spring-cloud-alibaba-examples/seata-example/account-service/src/main/resources/application.yml]: 这里为什么修改了默认密码

Comment (yuluo-yx) [reply spring-cloud-alibaba-examples/seata-example/account-service/src/main/resources/application.yml]: > 这里为什么修改了默认密码

测试 example 忘了改回去了，我晚上修改之后重新 push下

Comment (yuluo-yx) [reply spring-cloud-alibaba-examples/seata-example/account-service/src/main/resources/application.yml]: done!

**Review by ruansheng8 — APPROVED** [review]
LGTM.

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**